Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -68,6 +68,9 @@ final class ManageViewPledgeViewController: UIViewController {
     _ = self.closeButton
       |> \.accessibilityLabel %~ { _ in Strings.Dismiss() }
       |> \.width .~ Styles.minTouchSize.width
+
+    _ = self.menuButton
+      |> \.accessibilityLabel %~ { _ in Strings.Manage_your_pledge() }
```

Comment:
I'm still wondering whether `Menu, button` would be better because you're already on a screen called `Manage your pledge` where you navigated by tapping `Manage` button 😄